src/dom-to-image-more.js

```diff
@@ -954,7 +987,7 @@
             sandbox.contentDocument.title = 'sandbox';
         }
         const defaultElement = document.createElement(tagName);
```

Comment:
In #104, I changed this line to (essentially) `sandbox.contentDocument.createElement(tagName);` because I think we need to be creating the element against the sandbox document NOT against the `document`...

Reply:
Yep. Another one for the merge conflict :)